src/components/pages/ImageClassification/ImageAnnotationModal/ImageAnnotationModalMap.js

```diff
@@ -422,11 +431,10 @@ const ImageAnnotationModalMap = ({
       (feature) => !feature.properties.isConfirmed,
     )
 
-    zoomToBounds(getBounds(nextUnconfirmedFeature))
-    map.current.once('idle', () => {
-      selectFeature(nextUnconfirmedFeature)
-    })
-  }, [getPointsGeojson, map, selectFeature, selectedPoint.id, zoomToBounds])
+    zoomToPaddedBounds(getBounds(nextUnconfirmedFeature))
+
+    selectFeature(nextUnconfirmedFeature)
```

Comment:
Do you need to use the return value from selectFeature?

Reply:
It might be useful in the future, but this one doesnt seem to have an issue with not showing the popup (that I noticed)